chapters/external-repository-identifiers.md

```diff
@@ -200,6 +239,17 @@ Examples:
 
 External documentation: [Software Heritage](https://docs.softwareheritage.org/devel/swh-model/persistent-identifiers.html)
 
+
+### F.4.1 gitbom <a name="gitbom"></a>
+
+Locator Format: gitoid":"<git object type>":"<hash algorithm>":"<hash value>
+	
+Contextual Example: gitoid:blob:sha1:261eeb9e9f8b2b4b0d119366dda99c6fd7d35c64
+
+External Documentation: <https://gitbom.dev/>
```

Comment:
Formatting consistent with existing sections of the Annex F
```suggestion
External Reference Site: <https://gitbom.dev/>
```

Reply:
This is a generic changes that should port into later SPDX versions. LGTM